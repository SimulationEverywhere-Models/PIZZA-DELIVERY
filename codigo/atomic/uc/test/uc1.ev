00:00:00:010 abrir_uc 1
00:00:08:000 atiende_uc 1
00:00:13:000 atiende_uc 2
00:00:20:000 atiende_uc 3
00:00:30:000 recibe_ent_uc 2
00:00:48:000 recibe_ent_uc 3
00:00:50:000 cerrar_uc 1
