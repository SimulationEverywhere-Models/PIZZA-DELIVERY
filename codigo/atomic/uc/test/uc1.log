Mensaje I / 00:00:00:000 / Root(00) para top(01)
Mensaje I / 00:00:00:000 / top(01) para uc(02)
Mensaje D / 00:00:00:000 / uc(02) / ... para top(01)
Mensaje D / 00:00:00:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:00:010 / Root(00) / abrir_uc /      1.00000 para top(01)
Mensaje X / 00:00:00:010 / top(01) / abrir_uc /      1.00000 para uc(02)
Mensaje D / 00:00:00:010 / uc(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:00:010 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:00:010 / Root(00) para top(01)
Mensaje * / 00:00:00:010 / top(01) para uc(02)
Mensaje Y / 00:00:00:010 / uc(02) / lista_ll_uc /      1.00000 para top(01)
Mensaje D / 00:00:00:010 / uc(02) / ... para top(01)
Mensaje Y / 00:00:00:010 / top(01) / lista_ll_uc /      1.00000 para Root(00)
Mensaje D / 00:00:00:010 / top(01) / ... para Root(00)
Mensaje X / 00:00:08:000 / Root(00) / atiende_uc /      1.00000 para top(01)
Mensaje X / 00:00:08:000 / top(01) / atiende_uc /      1.00000 para uc(02)
Mensaje D / 00:00:08:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:08:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:08:000 / Root(00) para top(01)
Mensaje * / 00:00:08:000 / top(01) para uc(02)
Mensaje Y / 00:00:08:000 / uc(02) / ocupado_uc /      1.00000 para top(01)
Mensaje D / 00:00:08:000 / uc(02) / 00:00:02:000 para top(01)
Mensaje Y / 00:00:08:000 / top(01) / ocupado_uc /      1.00000 para Root(00)
Mensaje D / 00:00:08:000 / top(01) / 00:00:02:000 para Root(00)
Mensaje * / 00:00:10:000 / Root(00) para top(01)
Mensaje * / 00:00:10:000 / top(01) para uc(02)
Mensaje Y / 00:00:10:000 / uc(02) / ll_error_uc /      1.00000 para top(01)
Mensaje D / 00:00:10:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje Y / 00:00:10:000 / top(01) / ll_error_uc /      1.00000 para Root(00)
Mensaje D / 00:00:10:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:10:000 / Root(00) para top(01)
Mensaje * / 00:00:10:000 / top(01) para uc(02)
Mensaje Y / 00:00:10:000 / uc(02) / libre_uc /      1.00000 para top(01)
Mensaje D / 00:00:10:000 / uc(02) / ... para top(01)
Mensaje Y / 00:00:10:000 / top(01) / libre_uc /      1.00000 para Root(00)
Mensaje D / 00:00:10:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:13:000 / Root(00) / atiende_uc /      2.00000 para top(01)
Mensaje X / 00:00:13:000 / top(01) / atiende_uc /      2.00000 para uc(02)
Mensaje D / 00:00:13:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:13:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:13:000 / Root(00) para top(01)
Mensaje * / 00:00:13:000 / top(01) para uc(02)
Mensaje Y / 00:00:13:000 / uc(02) / ocupado_uc /      1.00000 para top(01)
Mensaje D / 00:00:13:000 / uc(02) / 00:00:02:000 para top(01)
Mensaje Y / 00:00:13:000 / top(01) / ocupado_uc /      1.00000 para Root(00)
Mensaje D / 00:00:13:000 / top(01) / 00:00:02:000 para Root(00)
Mensaje * / 00:00:15:000 / Root(00) para top(01)
Mensaje * / 00:00:15:000 / top(01) para uc(02)
Mensaje Y / 00:00:15:000 / uc(02) / encolar_p /      2.00000 para top(01)
Mensaje D / 00:00:15:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje Y / 00:00:15:000 / top(01) / encolar_p /      2.00000 para Root(00)
Mensaje D / 00:00:15:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:15:000 / Root(00) para top(01)
Mensaje * / 00:00:15:000 / top(01) para uc(02)
Mensaje Y / 00:00:15:000 / uc(02) / libre_uc /      1.00000 para top(01)
Mensaje D / 00:00:15:000 / uc(02) / ... para top(01)
Mensaje Y / 00:00:15:000 / top(01) / libre_uc /      1.00000 para Root(00)
Mensaje D / 00:00:15:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:20:000 / Root(00) / atiende_uc /      3.00000 para top(01)
Mensaje X / 00:00:20:000 / top(01) / atiende_uc /      3.00000 para uc(02)
Mensaje D / 00:00:20:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:20:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:20:000 / Root(00) para top(01)
Mensaje * / 00:00:20:000 / top(01) para uc(02)
Mensaje Y / 00:00:20:000 / uc(02) / ocupado_uc /      1.00000 para top(01)
Mensaje D / 00:00:20:000 / uc(02) / 00:00:02:000 para top(01)
Mensaje Y / 00:00:20:000 / top(01) / ocupado_uc /      1.00000 para Root(00)
Mensaje D / 00:00:20:000 / top(01) / 00:00:02:000 para Root(00)
Mensaje * / 00:00:22:000 / Root(00) para top(01)
Mensaje * / 00:00:22:000 / top(01) para uc(02)
Mensaje Y / 00:00:22:000 / uc(02) / encolar_p /      3.00000 para top(01)
Mensaje D / 00:00:22:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje Y / 00:00:22:000 / top(01) / encolar_p /      3.00000 para Root(00)
Mensaje D / 00:00:22:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:22:000 / Root(00) para top(01)
Mensaje * / 00:00:22:000 / top(01) para uc(02)
Mensaje Y / 00:00:22:000 / uc(02) / libre_uc /      1.00000 para top(01)
Mensaje D / 00:00:22:000 / uc(02) / ... para top(01)
Mensaje Y / 00:00:22:000 / top(01) / libre_uc /      1.00000 para Root(00)
Mensaje D / 00:00:22:000 / top(01) / ... para Root(00)
Mensaje X / 00:00:30:000 / Root(00) / recibe_ent_uc /      2.00000 para top(01)
Mensaje X / 00:00:30:000 / top(01) / recibe_ent_uc /      2.00000 para uc(02)
Mensaje D / 00:00:30:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:30:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:30:000 / Root(00) para top(01)
Mensaje * / 00:00:30:000 / top(01) para uc(02)
Mensaje Y / 00:00:30:000 / uc(02) / ocupada_ent_uc /      1.00000 para top(01)
Mensaje D / 00:00:30:000 / uc(02) / 00:00:00:030 para top(01)
Mensaje Y / 00:00:30:000 / top(01) / ocupada_ent_uc /      1.00000 para Root(00)
Mensaje D / 00:00:30:000 / top(01) / 00:00:00:030 para Root(00)
Mensaje * / 00:00:30:030 / Root(00) para top(01)
Mensaje * / 00:00:30:030 / top(01) para uc(02)
Mensaje Y / 00:00:30:030 / uc(02) / dist_out_uc /      2.00000 para top(01)
Mensaje D / 00:00:30:030 / uc(02) / 00:00:00:000 para top(01)
Mensaje Y / 00:00:30:030 / top(01) / dist_out_uc /      2.00000 para Root(00)
Mensaje D / 00:00:30:030 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:30:030 / Root(00) para top(01)
Mensaje * / 00:00:30:030 / top(01) para uc(02)
Mensaje Y / 00:00:30:030 / uc(02) / libre_uc_ent /      1.00000 para top(01)
Mensaje D / 00:00:30:030 / uc(02) / ... para top(01)
Mensaje Y / 00:00:30:030 / top(01) / libre_uc_ent /      1.00000 para Root(00)
Mensaje D / 00:00:30:030 / top(01) / ... para Root(00)
Mensaje X / 00:00:48:000 / Root(00) / recibe_ent_uc /      3.00000 para top(01)
Mensaje X / 00:00:48:000 / top(01) / recibe_ent_uc /      3.00000 para uc(02)
Mensaje D / 00:00:48:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:48:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:48:000 / Root(00) para top(01)
Mensaje * / 00:00:48:000 / top(01) para uc(02)
Mensaje Y / 00:00:48:000 / uc(02) / ocupada_ent_uc /      1.00000 para top(01)
Mensaje D / 00:00:48:000 / uc(02) / 00:00:00:030 para top(01)
Mensaje Y / 00:00:48:000 / top(01) / ocupada_ent_uc /      1.00000 para Root(00)
Mensaje D / 00:00:48:000 / top(01) / 00:00:00:030 para Root(00)
Mensaje * / 00:00:48:030 / Root(00) para top(01)
Mensaje * / 00:00:48:030 / top(01) para uc(02)
Mensaje Y / 00:00:48:030 / uc(02) / dist_out_uc /      3.00000 para top(01)
Mensaje D / 00:00:48:030 / uc(02) / 00:00:00:000 para top(01)
Mensaje Y / 00:00:48:030 / top(01) / dist_out_uc /      3.00000 para Root(00)
Mensaje D / 00:00:48:030 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:48:030 / Root(00) para top(01)
Mensaje * / 00:00:48:030 / top(01) para uc(02)
Mensaje Y / 00:00:48:030 / uc(02) / libre_uc_ent /      1.00000 para top(01)
Mensaje D / 00:00:48:030 / uc(02) / ... para top(01)
Mensaje Y / 00:00:48:030 / top(01) / libre_uc_ent /      1.00000 para Root(00)
Mensaje D / 00:00:48:030 / top(01) / ... para Root(00)
Mensaje X / 00:00:50:000 / Root(00) / cerrar_uc /      1.00000 para top(01)
Mensaje X / 00:00:50:000 / top(01) / cerrar_uc /      1.00000 para uc(02)
Mensaje D / 00:00:50:000 / uc(02) / 00:00:00:000 para top(01)
Mensaje D / 00:00:50:000 / top(01) / 00:00:00:000 para Root(00)
Mensaje * / 00:00:50:000 / Root(00) para top(01)
Mensaje * / 00:00:50:000 / top(01) para uc(02)
Mensaje Y / 00:00:50:000 / uc(02) / cerrado_uc /      1.00000 para top(01)
Mensaje D / 00:00:50:000 / uc(02) / ... para top(01)
Mensaje Y / 00:00:50:000 / top(01) / cerrado_uc /      1.00000 para Root(00)
Mensaje D / 00:00:50:000 / top(01) / ... para Root(00)
