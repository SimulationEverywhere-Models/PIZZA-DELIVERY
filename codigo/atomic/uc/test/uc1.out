00:00:00:010 lista_ll_uc 1
00:00:08:000 ocupado_uc 1
00:00:10:000 ll_error_uc 1
00:00:10:000 libre_uc 1
00:00:13:000 ocupado_uc 1
00:00:15:000 encolar_p 2
00:00:15:000 libre_uc 1
00:00:20:000 ocupado_uc 1
00:00:22:000 encolar_p 3
00:00:22:000 libre_uc 1
00:00:30:000 ocupada_ent_uc 1
00:00:30:030 dist_out_uc 2
00:00:30:030 libre_uc_ent 1
00:00:48:000 ocupada_ent_uc 1
00:00:48:030 dist_out_uc 3
00:00:48:030 libre_uc_ent 1
00:00:50:000 cerrado_uc 1
